plutus-streaming/src/Plutus/Streaming.hs

```diff
@@ -71,8 +71,10 @@ withChainSyncEventStream socketPath networkId point consumer = do
             localNodeSocketPath = socketPath
           }
 
-      -- FIXME this comes from the config file but Cardano.Api does not expose readNetworkConfig!
-      epochSlots = EpochSlots 40
+      -- This a parameter needed only for the Byron era. Since the Byron
+      -- era is over and the parameter has never changed it is ok to
+      -- hardcode this.
```

Comment:
:+1:

Reply:
Added a reference to `Cardano.Api.ConsensusModeParams` :+1: